File: c-001/main.go
package main

func Sum35(max int) int {
	sum := 0
	for i := 0; i < max; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}
	return sum
}

func main() {
	println(Sum35(1000))
}

File: c-001/main.php
<?php
function sum35($max) {
    $array = range(0, $max - 1);
    $array = array_filter($array, fn($a) => $a % 3 == 0 or $a % 5 == 0);
    return array_sum($array);
}

print sum35(1000);
